node-repository/src/test/java/com/yahoo/vespa/hosted/provision/restapi/v2/responses/node5.json

```diff
@@ -6,14 +6,13 @@
   "hostname": "host5.yahoo.com",
   "parentHostname": "dockerhost2.yahoo.com",
   "openStackId": "node5",
-  "flavor": "docker",
-  "canonicalFlavor": "docker",
+  "flavor": "[vcpu: 1.0, memory: 1.0 Gb, disk 100.0 Gb]",
```

Comment:
This is used (among others) as dimension for metrics, so may break some graphs @yngveaasheim

Reply:
If we have graphs where *node* (not host) flavor is a dimension we'll have to change them when this goes into use.